gen3/lib/aws.sh

```diff
@@ -334,6 +336,16 @@ EOM
   fi
 
 
+  # else ...
+  if [[ "$GEN3_WORKSPACE" =~ _qualysvm$ ]]; then
+      commonsName=${GEN3_WORKSPACE//_es/}
```

Comment:
Are you sure that's what you want to replace? shouldn't it be 
`commonsName=${GEN3_WORKSPACE//_qualysvm/}` instead